packages/trader/src/Modules/Trading/Containers/chart-widgets.jsx

```diff
@@ -55,8 +55,8 @@ export const ChartTopWidgets = connect(({ modules, ui }) => ({
     theme: ui.is_dark_mode_on ? 'dark' : 'light',
 }))(({ onSymbolChange, charts_ref, theme, is_digits_widget_active, active_category, open }) => {
     let yAxiswidth;
-    if (charts_ref && charts_ref.chart) {
-        yAxiswidth = charts_ref.chart.yAxiswidth;
+    if (charts_ref?.current?.mainStore?.chart) {
+        yAxiswidth = charts_ref.current.mainStore.chart.yAxiswidth;
```

Comment:
A question, is it possible to remove if here?
```suggestion
        yAxiswidth = charts_ref?.current?.mainStore?.chart.yAxiswidth;
```